test/el_cl_launcher_test.star

```diff
@@ -0,0 +1,153 @@
+el_cl_launcher = import_module("/src/el_cl_launcher.star")
+input_parser = import_module("/src/package_io/input_parser.star")
+observability = import_module("/src/observability/observability.star")
+ethereum_package_constants = import_module(
+    "github.com/ethpandaops/ethereum-package/src/package_io/constants.star"
+)
+util = import_module("/src/util.star")
+
+
+def test_launch_with_defaults(plan):
+    parsed_input_args = input_parser.input_parser(
+        plan,
+        {
+            "chains": [
+                {
+                    "participants": [
+                        {
+                            "el_type": "op-reth",
+                            "el_image": "op-reth:latest",
+                            "cl_type": "op-node",
+                            "cl_image": "op-node:latest",
+                        }
+                    ]
+                }
+            ]
+        },
+    )
+
+    observability_helper = observability.make_helper(parsed_input_args.observability)
+
+    deployment_output = "/path/to/deployment_output"
+    l1_config_env_vars = {
+        "L1_RPC_URL": "L1_RPC_URL",
+        "L1_RPC_KIND": "standard",
+        "CL_RPC_URL": "CL_RPC_URL",
+    }
+    jwt_file = "/path/to/jwt_file"
+    chains = parsed_input_args.chains
+    chain = chains[0]
+    da_server_context = struct(
+        enabled=False,
+        http_url="da_server_http_url",
+    )
+
+    # We'll mock read_network_config_value since it returns a runtime value that we would not be able to retrieve
+    sequencer_private_key_mock = "sequencer_private_key"
+    read_network_config_value_mock = kurtosistest.mock(
```

Comment:
`read_network_config_value_mock` doesn't seem to  be read afterwards. Should we have some expectations on it? or do we only want the side-effects of kurtosistest.mock ?

That makes me wonder, what's the scope for the mock? is it scoped to the current test?

Reply:
Oh yeah I can delete the mock variable.

In general everything is scoped to one test run. Every test function is ran in isolation so nothing is being kept between tests.